README.md

```diff
@@ -12,12 +12,20 @@ Javascript, Lua, Ocaml, Pascal, Perl, PHP, Prolog, Python, Ruby, Rust, Scheme.
 A markdown description of the input in English and French can also be
 generated.
 
+## Installation
+
+Depends on python 3.5+
```

Comment:
We use 3.7 typing.

Reply:
Everything works indeed with python 3.5. I test everything with a Debian stable, which has python 3.5.